### ISSUE TITLE
Make code blocks easier to read

### ISSUE THREAD

**Issue by benjifisher** [isovera/reveal.js#3]
Code blocks using triple backticks and a language identifier are displayed with colored text on a black  background, and they are hard to read. For example, this screenshot shows markdown (`.md`) rendered as dark blue on black:

<img width="1313" alt="revealjs-code-md" src="https://user-images.githubusercontent.com/1170799/54731119-6aa74e00-4b5a-11e9-82f6-3edef61465ca.png">

Let's use a white background for these code blocks instead of black.

**Comment by benjifisher** [comment]
Let's postpone this issue until after #4.
